orbit/pkg/update/runner.go

```diff
@@ -30,6 +30,17 @@ type Runner struct {
 	localHashes map[string][]byte
 }
 
+// UpdateRunnerOptTargets updates the RunnerOptions.Targets with the given target
+func (r *Runner) UpdateRunnerOptTargets(target string) {
```

Comment:
Looks like there's a race now.

We should now protect `r.opt.Targets` with a lock.

Reply:
updated, now using a mutex